Custom properties added to the Information tab - "working with..." chapters (#9078)
qgis/QGIS-Documentation: docs/user_manual/working_with_vector/vector_properties.rst

* general such as name in the project, source path, list of auxiliary files,
  last save time and size, the used provider

> * custom properties, used to store additional information about certain objects, such as map layers.

Reviewer: ```suggestion
* custom properties, used to store additional information about layers, in the active project.
  Default custom properties may include :ref:`layer notes <layer_notes>`,
  :ref:`legend widgets <vectorlegendmenu>`, :ref:`layer variables <general_tools_variables>`,
  form properties...
```
This is a specific property of the layers (not any objects) and we are already in a "dedicated chapter".
Not sure if it is clear enough that these properties are kept in the project, and no idea what the form view prop is.

>   These properties are created and managed using PyQGIS, specifically through the `setCustomProperty` method.

Reviewer: ```suggestion
  More custom properties can be created and managed using PyQGIS,
  specifically through the :pyqgis:`setCustomProperty() <qgis.core.QgsMapLayer.setCustomProperty>` method.
```
Adjusting a bit, given that this is not the only way to add custom properties.
